### PR TITLE
misc changes, part ?/? 

### PR DESCRIPTION
**Pull request by dima-xd**
Saw some games were using these calls

### CODE REVIEW

**Comment by gandalfthewhite19890404** [comment]
Used at least by:
Ghost Song - CUSA33533
Surviving The Aftermath - CUSA15495

File: src/core/libraries/videoout/video_out.cpp [hunk 1]
```diff
@@ -151,6 +151,15 @@ s32 PS4_SYSV_ABI sceVideoOutSubmitFlip(s32 handle, s32 bufferIndex, s32 flipMode
     return ORBIS_OK;
 }
 
+int PS4_SYSV_ABI sceVideoOutGetEventId(const Kernel::SceKernelEvent* ev) {
```
Comment (psucien) [inline src/core/libraries/videoout/video_out.cpp:154]: This should return `INVALID_EVENT` if not a VO event passed

File: src/core/libraries/videoout/video_out.cpp [hunk 2]
```diff
@@ -151,6 +151,15 @@ s32 PS4_SYSV_ABI sceVideoOutSubmitFlip(s32 handle, s32 bufferIndex, s32 flipMode
...
+    return ev->ident;
+}
+
+int PS4_SYSV_ABI sceVideoOutGetEventData(const Kernel::SceKernelEvent* ev, int64_t* data) {
+    *data = ev->data;
```
Comment (psucien) [inline src/core/libraries/videoout/video_out.cpp:172]: Lack of check for address validness and the same as above

**Review by psucien — CHANGES_REQUESTED** [review]

**Comment by raphaelthegreat** [comment]
What about this?

**Comment by dima-xd** [comment]
> What about this?

This too

**Pull request merged**